Scala/Scala.sublime-syntax

```diff
@@ -98,11 +99,17 @@ contexts:
       captures:
         1: keyword.declaration.scala
         2: entity.name.type.declaration
-    - match: '\b(?:(val)|(var))\s+(([a-zA-Z$_][a-zA-Z0-9$_]*(_[^a-zA-Z0-9\s]+)?)|`.*`)?'
+    - match: '\b(?:(var))\s+(([a-zA-Z$_][a-zA-Z0-9$_]*(_[^a-zA-Z0-9\s]+)?)|`.*`)?'
```

Comment:
The non-capturing group around `var` seems unnecessary: `\b(var)\s+` should be fine.